docs/user/config-file/v2.rst

```diff
@@ -418,6 +418,42 @@ We don't currently support PPA's or other custom repositories.
    When possible avoid installing Python packages using apt (``python3-numpy`` for example),
    :ref:`use pip or Conda instead <guides/reproducible-builds:pinning dependencies>`.
 
+
+build.jobs
+``````````
+
+Commands to be run before or after a Read the Docs pre-defined job.
+This allows you to run custom commands at a particular moment in the build process.
+
+
+.. code-block:: yaml
+
+   version: 2
+
+   build:
+     os: ubuntu-22.04
+     tools:
+       python: "3.10"
+     jobs:
+       pre_create_environment:
+         - echo "Command run at 'pre_create_environment' step"
+       post_build:
+         - echo "Command run at 'post_build' step"
+         - echo `date`
+
+.. note::
+
+   Each key under ``build.jobs`` must be a list of strings.
```

Comment:
I don't really know what this means. Can we communicate it more explicitly somehow? What else would users try to do here, that might be wrong?

Reply:
I'm trying to avoid users writing something like

```yaml
build:
  jobs:
    post_install: pip install mypackage
```

when it should be 

```yaml
build:
  jobs:
    post_install: 
      - pip install mypackage
```